polyaxon/scheduler/spawners/templates/env_vars.py

```diff
@@ -34,6 +34,8 @@ def get_service_env_vars(namespace='default'):
         get_from_app_secret('POLYAXON_INTERNAL_SECRET_TOKEN', 'polyaxon-internal-secret-token'),
         get_from_app_secret('POLYAXON_RABBITMQ_PASSWORD', 'rabbitmq-password',
                             settings.POLYAXON_K8S_RABBITMQ_SECRET_NAME),
+        get_from_app_secret('POLYAXON_DB_PASSWORD', 'postgres-password',
```

Comment:
This is the right way to do it, the in-cluster DB was basically using the default values which was a bad idea. I hope that other people did not go through that, it would be hard to debug, e.g. changing the password.